__tests__/app.test.js

```diff
@@ -74,6 +74,44 @@ describe("/api/reviews", () => {
           expect(body.review.review_id).toBe(review_id);
         });
     });
+    describe("/api/reviews/:reviews_id/comments", () => {
+      test("GET - 200 responds with an array of comments", () => {
+        const review_id = 2;
+        return request(app)
+          .get("/api/reviews/" + review_id +"/comments")
+          .expect(200)
+          .then((res) => {
+            expect(Array.isArray(res.body)).toBe(true);
+            expect(res.body).toHaveLength(3);
+            res.body.forEach((review) => {
+              expect(review).toEqual(
+                expect.objectContaining({
```

Comment:
We can be much more specific about what we expect back here rather than expect.any(String) etc. we can expect the exact contents of the comment we're requesting - check the comments test data and use that. This way we're not just testing the structure but the contents.